vms/vm-under-test/scripts/customize-vm

```diff
@@ -109,6 +107,8 @@ if [ ! -f "$checkup_tuned_adm_set_marker_full_path" ]; then
   exit 0
 fi
 
+boot_script_readiness_marker_full_path="/tmp/realtime-checkup-ready-marker"
```

Comment:
Since this variable is only used once, please consider in-lining it.

Reply:
This variable is going to be moved to a const once we move it to cloud-init. Keeping it will make the "copy" easier to understand. But I won't insist.